bitrix/admin/codecraft/forum_review.php

```diff
@@ -0,0 +1,182 @@
+<?
+/**
```

Comment:
Need to comment file version, developer's tag and description for business logic.